packages/cli/cli/src/cli-context/upgrade-utils/getFernUpgradeMessage.ts

```diff
@@ -0,0 +1,162 @@
+    if (limit != null) {
+        generatorsNeedingUpgrades = generatorsNeedingUpgrades.slice(0, limit + 1);
+    }
+    for (const generatorUpgrade of generatorsNeedingUpgrades) {
+        // Format => Generator 1.0.0 → 1.1.0 (API: API Name, Group: Group Name)
+        // ex: "Python SDK 1.0.0 → 1.1.0 (API: myApi, Group: myGroup)"
+        message +=
+            `\n${normalizeGeneratorName(generatorUpgrade.generatorName)} (${
+                generatorUpgrade.apiName != null ? "API: " + generatorUpgrade.apiName + ", " : ""
+            }Group: ${generatorUpgrade.generatorGroup}) ` +
+            chalk.dim(generatorUpgrade.currentVersion) +
+            chalk.reset(" → ") +
+            chalk.green(generatorUpgrade.latestVersion);
+
+        generatorUpgradeNeeded = true;
+    }
+
+    message += "\n";
+
+    if (generatorUpgradeNeeded) {
+        return includeBoxen
+            ? boxen(message, {
+                  padding: 1,
+                  float: "center",
+                  textAlignment: "center",
+                  borderColor: "yellow",
+                  borderStyle: "round"
+              })
+            : message;
+    }
+    return;
+}
+
+function normalizeGeneratorName(generatorName: string): string {
+    if (generatorName.startsWith("fernapi/")) {
+        generatorName = generatorName.replace("fernapi/", "");
+    }
+    switch (generatorName) {
+        // Python
+        case "fern-python-sdk":
+            return "Python SDK";
+        case "fern-pydantic-model":
+            return "Pydantic Model";
+        case "fern-fastapi-server":
+            return "FastAPI";
+        // TypeScript
+        case "fern-typescript-browser-sdk":
+        case "fern-typescript-node-sdk":
+        case "fern-typescript-sdk":
+            return "TypeScript SDK";
+        case "fern-typescript-express":
+            return "Express";
+        // Java
+        case "fern-java-sdk":
+            return "Java SDK";
+        case "java-model":
+            return "Java Model";
+        case "fern-java-spring":
+            return "Spring";
+        // Go
+        case "fern-go-sdk":
+            return "Go SDK";
+        case "fern-go-model":
+            return "Go Model";
+        case "fern-go-fiber":
+            return "Fiber";
+        // C#
+        case "fern-csharp-sdk":
+            return "C# SDK";
+        case "fern-csharp-model":
+            return "C# Model";
+        // Ruby
+        case "fern-ruby-sdk":
+            return "Ruby SDK";
+        case "fern-ruby-model":
+            return "Ruby Model";
+        // Misc.
+        case "fern-postman":
+            return "Postman";
+        case "fern-openapi":
+            return "OpenAPI";
+
+        default: {
+            if (generatorName.startsWith("fern-")) {
+                return generatorName.replace("fern-", "");
+            }
+            return generatorName;
+        }
+    }
+}
```

Comment:
I know you recently added display name so would be nice to delete this and leverage that here

Reply:
Yup can do!